controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ContactInformationMaintainer.java

```diff
@@ -29,28 +36,60 @@
     private static final Logger log = Logger.getLogger(ContactInformationMaintainer.class.getName());
 
     private final Organization organization;
+    private final String apiName;
 
-    public ContactInformationMaintainer(Controller controller, Duration interval, JobControl jobControl, Organization organization) {
+    public ContactInformationMaintainer(Controller controller, Duration interval, JobControl jobControl, Organization organization, String apiName) {
```

Comment:
Should we also get tenant info from the controller? I.e avoid the controller instance altogether? Or just comment that this is the probable end goal.